src/libraries/System.IO.FileSystem.Watcher/tests/Utility/FileSystemWatcherTest.cs

```diff
@@ -461,6 +461,11 @@ protected static bool CanCreateSymbolicLinks
 
         public static bool CreateSymLink(string targetPath, string linkPath, bool isDirectory)
         {
+            if (OperatingSystem.IsIOS() || OperatingSystem.IsTvOS() || OperatingSystem.IsMacCatalyst()) // OSes that don't support Process.Start()
```

Comment:
Android and Browser can't start processes either:

```suggestion
            if (OperatingSystem.IsIOS() || OperatingSystem.IsTvOS() || OperatingSystem.IsMacCatalyst() || OperatingSystem.IsAndroid() || OperatingSystem.IsBrowser()) // OSes that don't support Process.Start()
```

Reply:
Shouldn't those platforms be specified with the attribute as the other three? 
https://github.com/dotnet/runtime/blob/4bdaa8e2fb06b1065d0e418d21abd80dfff00a8f/src/libraries/System.Diagnostics.Process/src/System/Diagnostics/Process.cs#L1206-L1209
cc @jeffhandley